### ISSUE TITLE
一些bug

### ISSUE THREAD

**Issue by ReFengs10** [Emily-Qu/optimizer#1]
1. /optimizer/nanoGPT/data/openwebtext-small/prepare.py 中的第11，12行是
`train_bin_file = os.path.join(out_dir, "train_3.bin")`
`val_bin_file = os.path.join(out_dir, "val_3.bin") `
Readme写的是**这样会创建train.bin和val.bin两个数据文件**，并且但是在运行`python train.py config/train_owt-small.py`的时候，train.py里面写的也是train.bin和val.bin，而不是train_3.bin和val_3.bin，直接按Readme做会报错。
2. Readme里写的多卡运行时
`torchrun --standalone --nproc_per_node=8 train.py config/train_openwebtext-small.py`
但是config里面实际上是train_owt-small.py，而不是train_openwebtext-small.py。
3. nanoGPT/train.py的第362~366行，
`while True:`
    `# determine and set the learning rate for this iteration`
    `lr = get_lr(iter_num) if decay_lr else learning_rate`
    `for param_group in optimizer.param_groups:`
        `param_group['lr'] = lr`
这里会导致muon优化器内的muon_lr也改变。
